sql/2021/pwa/top_manifest_properties.sql

```diff
@@ -0,0 +1,92 @@
+#standardSQL
+# Top manifest properties
+
+CREATE TEMP FUNCTION getManifestProps(manifest STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+try {
+  manifestJSON = Object.values(JSON.parse(manifest))[0];
+  if (typeof manifestJSON === 'string') {
+    return null;
+  }
+  return Object.keys(manifestJSON);
+} catch {
+  return null
+}
+''';
+
+SELECT
+  "PWA Sites" AS type,
+  _TABLE_SUFFIX AS client,
+  property,
+  COUNT(0) AS freq,
+  total,
```

Comment:
```suggestion
  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
```

Reply:
As per above can't use this shortcut here due to `UNNEST` and `HAVING` clauses.